objects/Editors/contained/SpiritEditor/script.lua

```diff
@@ -12,24 +13,32 @@ function onLoad()
         width = 0,
         height = 0,
     })
+    Color.Add("SoftBlue", Color.new(0.45,0.6,0.7))
     Wait.time(scan, 0.5, -1)
     rescan = false
 end
 
 function scan()
     local objs = upCast(self, 0.4, 0.1, {"Tile"})
-    if #objs == 0 then
-        clearButtons()
-        return
+    local spiritBoard
+    for _, hit in pairs(objs) do
+        if hit.getVar("elements") ~= nil or hit.hasTag("Any") then
```

Comment:
elements are now the "Generic" type so they shouldn't be being returned here anymore